Monika After Story/game/script-ch30.rpy

```diff
@@ -520,26 +520,32 @@ label ch30_autoload:
     #Run actions for any events that are based on the clock
     $ evhand.event_database=Event.checkCalendar(evhand.event_database)
 
-    #Skip all greetings if you closed the game on Monika
-    if persistent.closed_self:
-        #pick a random greeting
-        if is_monika_in_room:
-            if persistent.current_monikatopic != "i_greeting_monikaroom":
-                $ pushEvent("i_greeting_monikaroom")
-        else:
-            $pushEvent(renpy.random.choice(greetings_list))
+    #if the player told Monika that they are sick, jump to this greeting.
+    if persistent.sick:
+        $ pushEvent("greeting_sick")
 
-    if not persistent.tried_skip:
-        $ config.allow_skipping = True
     else:
-        $ config.allow_skipping = False
+        #Skip all greetings if you closed the game on Monika
+        if persistent.closed_self:
+            #pick a random greeting
+            if is_monika_in_room:
+                if persistent.current_monikatopic != "i_greeting_monikaroom":
+                    $ pushEvent("i_greeting_monikaroom")
+            else:
+                $pushEvent(renpy.random.choice(greetings_list))
 
-    if not is_monika_in_room:
-        $ set_keymaps()
+        if not persistent.tried_skip:
+            $ config.allow_skipping = True
+        else:
+            $ config.allow_skipping = False
 
-    $persistent.closed_self = False
-    $startup_check = False
-    jump ch30_loop
+        if not is_monika_in_room:
+            $ set_keymaps()
+
+        $persistent.closed_self = False
+    
+        $startup_check = False
+        jump ch30_loop
```

Comment:
Lines 542-548 also need to run regardless of greeting/no greeting, bring theses down an indentation level.